### PR TITLE
Added Team meeting for nov16

### PR DESCRIPTION
**Pull request by aleksssk**
Team meeting for nov16

### CODE REVIEW

File: meeting-minutes/team-minutes/nov16-team-minutes.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Meeting Minutes - November 16th
+
+## Agenda Items:
+- 「Aleks」Reminder that I will not be able to attend meeting on 23rd
+- 「Aleks」Reminder that meeting on The 30th is now on 23rd
```
Comment (jpabadir) [inline meeting-minutes/team-minutes/nov16-team-minutes.md]: I think you meant 29th?

**Review by jpabadir — APPROVED** [review]
Approved, but please address my comment before merging.

**Review by wrc9817 — APPROVED** [review]

File: meeting-minutes/team-minutes/nov16-team-minutes.md (hunk 1 above)
Comment (katmartel) [reply meeting-minutes/team-minutes/nov16-team-minutes.md]: @aleksssk can you clarify this before I approve?

Comment (aleksssk) [reply meeting-minutes/team-minutes/nov16-team-minutes.md]: Fixed! thanks for letting me know

**Review by katmartel — APPROVED** [review]

**Pull request merged**